fornax/opt.py

```diff
@@ -103,20 +103,17 @@ class Base(np.recarray):
     types = []
 
```

Comment:
Here I've updated the `__new__` method to create a mixed type recarray from a list of lists without ever converting the values to floating point.